hack/update-build-definitions.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+# Copyright 2023 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Updates a local clone of redhat-appstudio/build-definitions to use the latest
+# Task bundle produced by this repository.
+# Usage:
+#   update-build-definitions.sh <PATH_TO_BUILD_DEFINITIONS> [<TAG>]
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)"
+
+TARGET_DIR="${1}"
+cd "${TARGET_DIR}" || exit 1
+
+echo 'Resolving task bundle...'
+TASK_BUNDLE_TAG="${2:-snapshot}"
+MANIFEST=$(mktemp --tmpdir)
+function cleanup() {
+    rm "${MANIFEST}"
+}
+trap cleanup EXIT
+skopeo inspect "docker://quay.io/hacbs-contract/ec-task-bundle:${TASK_BUNDLE_TAG}" --raw > "${MANIFEST}"
+TASK_BUNDLE_DIGEST="$(skopeo manifest-digest "${MANIFEST}")"
+REVISION="$(jq -r '.annotations["org.opencontainers.image.revision"]' "${MANIFEST}")"
+if [[ -n "${REVISION}" && "${REVISION}" != null ]]; then
+    TASK_BUNDLE_TAG="${REVISION}"
+fi
+# Sanity check
+diff \
+    <(skopeo inspect --raw "docker://quay.io/hacbs-contract/ec-task-bundle:${TASK_BUNDLE_TAG}") \
+    <(skopeo inspect --raw "docker://quay.io/hacbs-contract/ec-task-bundle@${TASK_BUNDLE_DIGEST}")
```

Comment:
Does it return non zero if there is a diff?

Reply:
Yes, quick example:

```
diff <(echo A) <(echo B) && echo not happening
```